### PR TITLE
Post release cleanup

### PR DESCRIPTION
**Pull request by Arau**

### CODE REVIEW

**Comment by darkowlzz** [comment]
@Arau can you also update the [kubernetes-csi docs](https://github.com/storageos/storageos.github.io/blob/master/_docs/install/kubernetes-csi/dynamic-provisioning.md#dynamic-provisioning-csi) as part of this? 
Right now it instructs to use the `csi-deployment` branch. But now, we can use the chart from master.

**Comment by Arau** [comment]
@darkowlzz I have added a change to work from master branch of the helm chart. Can you double check that it was what you meant, please?

**Comment by darkowlzz** [comment]
@Arau yes, that looks good. I added two small changes and pushed in the same commit.

File: _docs/install/cloud/azure.md [hunk 1]
```diff
@@ -115,7 +115,9 @@ Running `kubectl cluster-info` should now connect to your new cluster.
 Consult the [acs-engine documentation](https://github.com/Azure/acs-engine/blob/master/docs/kubernetes/deploy.md)
 for more information.
 
-### StorageOS installation
+### StorageOS installation with Helm
+
+>If you want to install StorageOS without Helm, check out the following [page]({%link _docs/install/kubernetes/index.md %}).
```
Comment (darkowlzz) [inline _docs/install/cloud/azure.md]: Maybe "To install StorageOS without Helm, ..." ?

File: _docs/reference/os_support.md [hunk 2]
```diff
@@ -29,8 +29,13 @@ The following table shows which distributions can enable the kernel modules to u
 
 ## How to enable LIO support
 
-The following script will try to load the modules in your host. If it finishes without error, your OS is compatible and LIO support will be enabled.
+The following script tries to load kernel modules in your host. 
+If it finishes without error, your OS is compatible and LIO support will be enabled.
 
+You can run the script or execute the containerised version of it. Both solutions require the execution for every host that will run StorageOS.
```
Comment (darkowlzz) [inline _docs/reference/os_support.md]: Maybe "execution on every host..."?

File: _docs/install/cloud/azure.md (hunk 1 above)
Comment (Arau) [reply _docs/install/cloud/azure.md]: done

Comment (Arau) [reply _docs/install/cloud/azure.md]: done

File: _docs/reference/os_support.md (hunk 2 above)
Comment (Arau) [reply _docs/reference/os_support.md]: done

**Pull request merged**